core/test/models/workarea/navigation/breadcrumbs_test.rb

```diff
@@ -23,10 +23,12 @@ def test_collection
       end
 
       def test_last
```

Comment:
After the suggestion above, you should be able to reduce this to just testing a navigable as `last`.